internal/services/forwards/mocks_test.go

```diff
@@ -3,35 +3,17 @@ package forwards
 import (
 	"context"
 	"io"
+
+	"github.com/ciph-r/postage/internal/traffic"
 )
 
 // ClientLoadBalancerMock satisfies traffic.ClientLoadBalancer interface.
 type ClientLoadBalancerMock struct {
-	GetClientConnectionFunc func(ctx context.Context, id string) (*ClientConnectionMock, error)
-}
+	traffic.LoadBalancer
```

Comment:
this is a hack to avoid mocking unused methods. 

it works similar to method overloading in other languages. its allows the struct to satisfy the interface without actually implementing the methods, or in this case, only implementing the methods we need. this pattern is really only acceptable in tests for ease of mocking.